### PR TITLE
fix html

### PR DESCRIPTION
**Pull request by bondrogeen**

### CODE REVIEW

File: admin/index_m.html [hunk 1]
```diff
@@ -1513,7 +1523,12 @@
                     </div>
                 </div>
             </div>
-            <div id="tab-smart-enums"  class="col s12 page">
+            <div id="tab-smart-enums" class="col s12 page">
+                <div class="row">
+                    <div class="col s6">
+                        <img src="cloud.png" class="logo">
```
Comment (WolfspiritM) [inline admin/index_m.html]: Is there a reason you're introducing the cloud logo on every page? It causes a very ugly grey bar. Especially in the smart devices tab:

image

**Comment by bondrogeen** [comment]
Well, tomorrow I will correct according to your requirements

**Comment by WolfspiritM** [comment]
Thank you 😄

**Comment by bondrogeen** [comment]
Just still you need to wait for the next update iobroker.admin, for the correct display.

**Pull request merged**